workitem/workitem_repository.go

```diff
@@ -594,7 +609,8 @@ func (r *GormWorkItemRepository) Save(ctx context.Context, spaceID uuid.UUID, up
 		}
 		fieldValue := updatedWorkItem.Fields[fieldName]
 		var err error
-		if fieldName == SystemAssignees || fieldName == SystemLabels || fieldName == SystemBoardcolumns {
+		// Remove fields with type list if there are no elements (eg: Assignee, Labels)
+		if fieldDef.Type.GetKind() == KindList {
```

Comment:
Thank you for making this change. Yesterday I've talked about this exact line with @xcoulon and we temporarily changed it to the exact line.

Reply:
You're welcome